src/file_path.rb

```diff
@@ -117,7 +117,7 @@ def rendered_build_file_path(rendered_dir, section, file_name)
       end
 
       def rendered_files
-        '/var/lib/metalware/rendered'
+        Constants::RENDERED_DIR_PATH
```

Comment:
As you have defined this method as a constant, it can be removed. `RENDERED_DIR_PATH` will be dynamically loaded in as `rendered_dir`

Reply:
Yep I missed that! Fixed